public/js/nicab.json

```diff
@@ -73,8 +73,8 @@
   },
   {
     "title": "Coleraine",
-    "address": "1-5 Brook Street\nColeraine\nLondonderry\nBT52 1PW",
-    "phone": "028 7032 9775",
+    "address": "24 Lodge Road\nColeraine\nBT52 1NB",
```

Comment:
I was getting the same coordinates..

```ruby
require 'geocoded'

address = '24 Lodge Road, Coleraine, BT52 1NB'
coordinates = Geocoder.coordinates(address)
=> [55.1315202, -6.6656869]
Geocoder.search(coordinates.join(",").first.address
=> "24 Lodge Rd, Coleraine, Coleraine BT52 1NB, UK"
```

The new address is [here in Google Earth](https://www.google.co.uk/maps/place/55°07'53.5%22N+6°39'56.5%22W/@55.1305355,-6.6686615,520m/data=!3m1!1e3!4m2!3m1!1s0x0:0x0) and the [old address](https://www.google.co.uk/maps/place/5+Brook+St,+Coleraine+BT52+1EN/@55.132578,-6.6671981,501m/data=!3m1!1e3!4m2!3m1!1s0x486024b55c1129e5:0xe4dc7c7d7041531a) is a couple of blocks away. The old address is @ 55.133378, -6.6671518 according to Google's geocoder. No idea why we had the old address but the new coordinates..